example/src/app/tealium/utag.service.ts

```diff
@@ -1,64 +1,68 @@
 import { Injectable } from '@angular/core';
 @Injectable()
 export class TealiumUtagService {
-  script_src: string = '';
- 
+  scriptSrc = '';
```

Comment:
Do we want to get rid of the 'string' type declaration here?

Reply:
tslint message on that line: Type string trivially inferred from a string literal, remove type annotation. 

It's recommended to remove it. could be left.